mtp_api/apps/transaction/fixtures/test_transactions.json

```diff
@@ -0,0 +1,1046 @@
+[
+{
+    "model": "transaction.transaction",
+    "fields": {
+        "amount": 29363,
+        "upload_counter": 1,
+        "prisoner_number": "PXAYGDQB41",
+        "created": "2015-06-16T13:20:46.378Z",
+        "prison": 2,
+        "modified": "2015-06-16T13:20:46.379Z",
+        "sender_customer_reference": "3S6PWQZ2NC",
+        "reference": "1KZ4V971CS",
+        "prisoner_dob": "1970-03-21",
+        "received_at": "2015-06-13T11:30:46.375Z",
+        "prisoner_name": "Florence Mckee",
+        "sender_bank_reference": "CARRM91UR7"
+    },
```

Comment:
something like `ZG4104H` is OK?
I will also correct the transaction reference at this point.
It's `<prisoner_number> <prisoner_dob>` eg. `Claire Muscara 01/09/1934`, right?